parkings/api/operator/parking.py

```diff
@@ -1,53 +1,22 @@
 from django.conf import settings
-from django.db import transaction
 from django.utils.timezone import now
 from django.utils.translation import ugettext_lazy as _
 from rest_framework import exceptions, permissions, serializers, viewsets
 
-from parkings.models import Address, Operator, Parking
+from parkings.models import Operator, Parking
 
-from ..common import AddressSerializer, ParkingFilter
+from ..common import ParkingFilter
 
 
 class OperatorAPIParkingSerializer(serializers.ModelSerializer):
     status = serializers.ReadOnlyField(source='get_state')
-    address = AddressSerializer(allow_null=True, required=False)
```

Comment:
Yeah, well, we will probably remove addresses completely, but that is not sure yet, so I started with removing it from Operator API, which is the one that actually matters ATM. :)